components/raftstore/src/store/fsm/store.rs

```diff
@@ -1324,6 +1324,8 @@ impl<EK: KvEngine, ER: RaftEngine> RaftBatchSystem<EK, ER> {
             warn!("set thread priority for raftstore failed"; "error" => ?e);
         }
         self.workers = Some(workers);
+        // This router will not be accessed again, free all caches.
+        self.router.clear_cache();
```

Comment:
Seems it's the main changes to fix the leak. Does the crossbeam update related to this fix?

Reply:
See https://github.com/crossbeam-rs/crossbeam/pull/669

Reply:
Seems this PR has not been included in the release version of crossbeam yet?

Reply:
Yes, that's the reason why a forked repository is used.

Reply:
`self.router`'s cache only contains 1024 slots, the main fix should be the crossbeam patch?

Reply:
Yes.